### DISCUSSION TITLE
2022-10-12 Weekly Meeting 77 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#302] [Weekly Meetings]
Weekly Meeting 77 is happening on Wednesday October 12, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).
We regularly use the # omi-meeting-chat channel for text chat. 

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda
- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Working Group Updates:
- [Scripting group](https://github.com/omigroup/omi-scripting-group)
  - OMI developer hour* 
- [glTF Extensions Group](https://github.com/omigroup/gltf-extensions)
  - Progress on OMI_ref and authoring extensions
- [Media Group](https://github.com/omigroup/media)
- [OMI Website update](https://github.com/omigroup/website)
- [MSF delegates](https://github.com/omigroup/MSF-Delegates) (setting up the repo)
- [M3 avatar interop](https://github.com/M3-org/avatar-interop#calendar)
- [UX Research: TBD](https://github.com/omigroup/ux-research) (setting up the repo) 
  - #222 repo is formed, when do we have the first meeting? https://github.com/omigroup/ux-research/issues/6
- any others?

### Discussions: 
- Open Metaverse conference
- What is our next demo?
- https://github.com/omigroup/omigroup/issues/301

### Future demo themes: (Vote now in the `#omi-vote` Discord channel!)
- `OMI-audio-emitter` exploration and demo 
- `OMI-collider` exploration and demo 
- `Generative and AI tooling`
- Any other suggestions?

**Comment by mrmetaverse** [comment]
# Developer Hour 
The group is considering doing a little world-building session during a future dev hour. 

Maybe we can do a field trip to visit different platforms, maybe run some tests, etc. 
- These have worked well on Fridays in the past. 
- Maybe some impromptu too? Anyone in the community can really just announce/host field trips.

**Comment by mrmetaverse** [comment]
# Dogfooding open standards 

- What if we start experimenting more, and showcasing how to use the extensions and protocols that are being created for and by the open Metaverse overall? 
- Maybe during a Wednesday, or a Dev hour we could dive in to what is needed, or how we could better delegate something like this. 
- We could put in a Call for participation. 
- Can we practically show - here is how to use `OMI-collider` and how it is being used and who is using it. 
- Maybe create an event. 
- Maybe create a proposal for what a role would look like to organize this sort of thing. 
- Maybe create and publish content related to the the protocol/extension of the week. 
-

**Comment by mrmetaverse** [comment]
# Seeking new volunteers for roles 
- Events 
- Financial Oversight (open collective and finances) 
- Video content / production
- (more, please reach out to Evo @LightLodges who will have to dial back some of their contributions )

**Comment by mrmetaverse** [comment]
# P1-OM demo day 

P1 are doing a live demo anyone is welcome to come. You can join their discord with this link: https://discord.gg/3JXWfT6V?event=1029080519226437773

**Comment by mrmetaverse** [comment]
# WebXR Production Summit is tomorrow 

You can learn more about the event with this link: https://webxr.events/event/webxr-production-summit/

**Comment by mrmetaverse** [comment]
# RE: Recordings 

- Are you listening to our recordings each week? 
- Do you find them useful? 
- Do you have an alternative?

**Reply by mrmetaverse** [reply]
"I am grateful for the recordings, and systems created by you (Shirley)".

**Reply by mrmetaverse** [reply]
I personally think the recordings are important for preservation purposes.

**Comment by mrmetaverse** [comment]
# Media group 
- see above 
- seeking volunteers

### Q: Is there a summary doc of the strategy for the media group? And opportunities to collaborate? 
**There is an attempt to build a better github page including:** 
- how to speak about OMI 
- how to participate in OMI

**Reply by mrmetaverse** [reply]
"maybe a workshop around our narrative strategy"

**Reply by mrmetaverse** [reply]
#301

**Comment by mrmetaverse** [comment]
# How can we make it easier to get involved? 
**What if we list our known gaps/roles.?**
- add openings to working group repo readmes 
- add to the website (github isn't always friendly for non-developers)

**Reply by mrmetaverse** [reply]
What the group is, who we are, what we do, etc. (currently handled by our README's but how can we be even more newbie-friendly?)

**Reply by mrmetaverse** [reply]
Let's learn from Sindhu

**Reply by mrmetaverse** [reply]
https://github.com/omigroup/omigroup/issues/303

**Comment by mrmetaverse** [comment]
# Events / Open Metaverse Conference 

Jamil is with us again today! 
"This is an event where we will want to show, not tell. We will also have a demo room, where we will invite people to demo. Demos are more engaging, where you can react and respond. Whereas a speech or keynote speaker requires a silent crowd."

**Reply by mrmetaverse** [reply]
Evo: "In the past we have structured our demos in different ways. Virtual participation is very important to this community"

**Reply by mrmetaverse** [reply]
Jamil: "One thing we can do is have a stage space for injecting OMI's process. It would be incredible if you could maintain that for two days. but if there are talks that you all want to have, we can incorporate those... there will be multiple tracks: creator, technology (developers), etc. "

**Reply by mrmetaverse** [reply]
f: "there is some interest in the idea of a 2 day experiment, but there are some challenges we have to address."

**Reply by mrmetaverse** [reply]
Question: "the idea of doing the extended demo series is part of the "official partner" requirements?"

"Actually, we are good to change the language from official partner to participating organizations"

**Reply by mrmetaverse** [reply]
We can resume this talk next week."

**Comment by mrmetaverse** [comment]
# UX Research group 
the group will kick off on Tuesday 10/18/22 at 16:00 UTC! SPREAD THE WORD <3
